js/views/composer.js

```diff
@@ -0,0 +1,369 @@
+/* global Backbone, Handlebars, models, OC, Mail, _ */
+
+var views = views || {};
+
+views.Composer = Backbone.View.extend({
+
+	type: 'new',
+
+	attachments: null,
+
+	submitCallback: null,
+	sentCallback: null,
+	draftCallback: null,
+
+	aliases: null,
+	accountId: null,
+	folderId: null,
+	messageId: null,
+
+	draftInterval: 1500,
+	draftTimer: null,
+	draftUID: null,
+	hasData: false,
+	autosized: false,
+
+	events: {
```

Comment:
Really nice! I wanted to get rid of all those jquery-fu in mail.js for long time! 

@ChristophWurst 

![youre-awesome](https://cloud.githubusercontent.com/assets/1005065/8475801/f355787c-20ba-11e5-8644-417c67afd614.jpg)